.pre-commit-config.yaml

```diff
@@ -10,6 +10,12 @@ repos:
     hooks:
       - id: identity
       - id: check-hooks-apply
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v3.18.0
+    hooks:
+      - id: pyupgrade
+        args: [--py36-plus]
+        exclude: ^python/sedona/sql/dataframe_api\.py$
```

Comment:
why is this excluded?

Reply:
Hey @jiayuasu I remember this file was causing problems when formatting.

I will review this again and report back.  Thanks